pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/client/tabs/PentahoTab.java

```diff
@@ -148,10 +148,6 @@ public String getLabelTooltip() {
   }
 
   protected void closeTab() {
-    if ( getContent().getElement().getElementsByTagName( "iframe" ).getLength() > 0 ) {
```

Comment:
I've add this event to MantleTabPanel:
https://github.com/pentaho/pentaho-platform/pull/2892